client/src/components/OverviewProfileCard.js

```diff
@@ -0,0 +1,50 @@
+import React, { useContext } from 'react';
+import { Card } from 'react-bootstrap';
+import styled from 'styled-components';
+import StyledButton from '../constant/StyledButton';
+import avatar from '../assets/icons/avatar.svg';
+import { ProfileContext } from '../context/ProfileContext';
+
+
+const OverviewProfileCard = ({ profile }) => {
+	const { getProfile } = useContext(ProfileContext);
+
+	const handleClick = (id)=>{
+		getProfile(id);
+	};
```

Comment:
If you want to load on page load, rather than the extra click of a button?
```suggestion
	const { getProfile } = useContext(ProfileContext);

	useEffect(
		() => getProfile(profile.id),
		[]
	);
```